runners/flink/runner/pom.xml

```diff
@@ -159,6 +159,19 @@
       </plugin>
       -->
 
+      <!-- Disable javadoc for now.
+           TODO: this section should be removed as soon as possible. -->
```

Comment:
I will double check as I'm not sure it's enough for the maven-release-plugin execution.